snapcraft/internal/meta.py

```diff
@@ -163,25 +164,28 @@ def write_snap_directory(self):
                     os.remove(destination)
                 file_utils.link_or_copy(source, destination)
 
-        # Now copy the hooks contained within the snap directory directly into
-        # meta (they don't get wrappers like the ones that come from parts).
-        snap_hooks_dir = os.path.join('snap', 'hooks')
-        hooks_dir = os.path.join(self._snap_dir, 'meta', 'hooks')
-        if os.path.isdir(snap_hooks_dir):
-            os.makedirs(hooks_dir, exist_ok=True)
-            for hook_name in os.listdir(snap_hooks_dir):
-                source = os.path.join(snap_hooks_dir, hook_name)
-                destination = os.path.join(hooks_dir, hook_name)
-
-                # First, verify that the hook is actually executable
-                if not os.stat(source).st_mode & stat.S_IEXEC:
-                    raise CommandError('hook {!r} is not executable'.format(
-                        hook_name))
-
-                with contextlib.suppress(FileNotFoundError):
-                    os.remove(destination)
-
-                file_utils.link_or_copy(source, destination)
+        # Now copy the assets contained within the snap directory directly into
+        # meta (they don't get wrappers for hooks like the ones that come from
+        # parts).
```

Comment:
This comment doesn't quite make sense changing 'hooks' to 'assets,' as assets has nothing to do with wrappers.